CHANGELOG.md

```diff
@@ -21,6 +21,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Compiling stuck in the browser for recursive zkprograms https://github.com/o1-labs/o1js/pull/1906
 
+### Added
+
+- Provide `zip` and `pad` for the array utility https://github.com/o1-labs/o1js/pull/1921
```

Comment:
nit: we actually dont expose the array utilities to the outside world directly, we use them as proxy most of the time - so these functions are just internal for now. I would remove the changelog entry but still merge the additions!